README.md

```diff
@@ -17,35 +17,38 @@ Features
 
 If the server has enabled client certificate authentication, first install the client certificate and private key to system keychain. You may first need to convert them to PKCS#12 format with something like, `openssl pkcs12 -export -in client.crt -inkey client.key -certfile ca.pem -out client.p12`. Otherwise, the program will perform server certificate validation only.
 
-Compiling Plan B requires a modern version of Xcode, available from Apple's developer site.
+Compiling Plan B requires a modern version of Xcode, available from Apple's Developer site.
 
-* Check out the code with `git clone https://github.com/google/macops` and open the Xcode project with `open macops/planb/planb.xcodeproj`
+* Download the source code with `git clone https://github.com/google/macops-planb` 
 
-By default, the program will request `https://mac.internal.megacorp.com/pkgs/pkg1/package1-stable.dmg`, `.../pkg2/package2-stable.dmg` and so on.
+* Open the Xcode project with `open macops-planb/planb.xcodeproj`
 
-* Edit `PBConnectionDelegate.m` and change `kConnectionDelegateHost` to the hostname of the server and `kConnectionDelegatePackageBase` to the path of the folder containing the disk images. By default, the program will use `https://mac.internal.megacorp.com/pkgs/`
+* Edit `PBURLBuilder.m` and change `kBaseURL` to the URL of the server and folder containing disk images. By default, the program will use `https://mac.internal.megacorp.com/pkgs/`
 
-* Edit `main.m` and change the `packages` array to match the names of the disk image names and their contained packages' receipt names. By default, the program will construct `pkg1/package1-stable.dmg` and forget the receipt for package `com.megacorp.package1` prior to re-installation, and so on.
+* Edit `main.m` and change the `packages` array to match the names of disk image names and their contained packages' receipt names. By default, the program will construct `pkg1/package1-stable.dmg` and forget the receipt for package `com.megacorp.package1` prior to re-installation, and so on.
 
 * Edit `PBURLBuilder.m` and change the `kMachineInfo` to match a machine information plist, which may contain a `ConfigurationTrack` value, for example. This value is used to construct the disk image suffix, like `package1-stable.dmg`, `package1-testing.dmg` or `package1-unstable.dmg`. This is useful if you have machines on multiple configuration tracks.
 
 * Edit `roots.pem` and change the contents to include a single or multiple PEM-encoded certificate authority certificates you wish to trust for server validation. By default, the program will use `GeoTrust Global CA`, the authority used to sign Google's intermediate CA, however you should use the CA which has signed the server's certificate or the server's intermediate certificate.
 
-* Build and run the program. It requires root privileges in order to install the software packages.
+* Install required Cocoa Pod(s) with `pod install`
```

Comment:
CocoaPods, one word. Also, this step should probably be directly after downloading the source and then the "open Xcode project" step can open the workspace instead.